frontend/integration-tests/views/crud.view.ts

```diff
@@ -131,11 +131,12 @@ export const deleteRow = (kind: string) => (name: string) =>
     return browser.wait(until.or(kebabIsDisabled, until.or(listIsEmpty, rowIsGone)));
   });
 
-export const rowFilters = $$('.row-filter__box');
-export const rowFiltersPresent = () => browser.wait(until.presenceOf($('.row-filter__box')));
-export const rowFilterFor = (name: string) =>
-  rowFilters.filter((el) => el.getText().then((text) => text.includes(name))).first();
-export const activeRowFilters = $$('.row-filter__box--active');
+export const rowFiltersButton = $('[data-test-id="filter-dropdown-toggle"]');
+export const rowFilterItems = $$('[data-test-id="filter-row-dropdown-item"]');
+export const rowFiltersPresent = () => browser.wait(until.presenceOf(rowFiltersButton));
+export const rowFilterFor = (name: string) => {
+  return rowFilterItems.filter((el) => el.getText().then((text) => text.includes(name))).first();
```

Comment:
I think this would be greatly simplified if we added the `data-test-row-filter` on the individual filter item like

```tsx
data-test-row-filter={text}
```

The this becomes

```ts
export const rowFilterFor = (name: string) => $(`[data-test-row-filter="${name}"`);
```

and you don't need `rowFilterItems`